ckanext/bcgov/templates/package/read.html

```diff
@@ -119,7 +119,7 @@
   {% block package_additional_info %}
     <h3>Additional Information</h3>
     <div id="additional-information">
-      {% if pkg.object_name and pkg.resources and pkg.resources[0].resource_storage_location == 'BCGW Data Store' %}
+      {% if pkg.object_name and pkg.resources %}
```

Comment:
Seems to have worked!

On Wed, Jun 3, 2015 at 9:13 AM, Greg Lawrance <greg.lawrance@gmail.com>
wrote:

> Yes - that is correct. If you put the object_name into a geographic
> pacakge it should display it as long as their is a data resource of one
> kind or another.
>
> On Wed, Jun 3, 2015 at 6:52 AM, Ian Ward <notifications@github.com> wrote:
>
>> In ckanext/bcgov/templates/package/read.html
>> <https://github.com/bcgov/ckanext-bcgov/pull/1#discussion_r31624173>:
>>
>> > @@ -119,7 +119,7 @@
>> >    {% block package_additional_info %}
>> >      <h3>Additional Information</h3>
>> >      <div id="additional-information">
>> > -      {% if pkg.object_name and pkg.resources and pkg.resources[0].resource_storage_location == 'BCGW Data Store' %}
>> > +      {% if pkg.object_name and pkg.resources %}
>>
>> So, this is the change that should be visible. I'll find a dataset with a
>> non-"BCGW Data Store" resource[0] and see if the change takes place.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/bcgov/ckanext-bcgov/pull/1/files#r31624173>.
>>
>
>